dataset_metrics/Dataset_metrics.ipynb

```diff
@@ -378,13 +376,40 @@
     "            kind=\"bar\")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "revolutionary-power",
+   "metadata": {},
+   "source": [
+    "### Number of articles with and without valid \"instance of\""
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "worse-fleece",
+   "id": "healthy-bacteria",
    "metadata": {},
    "outputs": [],
-   "source": []
+   "source": [
+    "query = \"\"\"SELECT wiki_db, snapshot,\n",
+    "        COUNT(instance_of) AS with_instance_of,\n",
+    "        SUM(CASE WHEN instance_of IS NULL then 1 ELSE 0 END) AS without_instance_of\n",
+    "        FROM gmodena.imagerec_parquet\n",
```

Comment:
We really need to move this data out of my user home dir.

In the meanwhile, it would be nice to template this query so that `snapshot` and table name can be passed are parameters.
Not a stopper for this PR, but maybe let's keep it as a TODO.

Reply:
+1 to moving it out of user home dir.

Added a todo for this.